plugins/slidev/skills/create-slide/references/animations.md

```
@@ -517,66 +517,66 @@ transition: my-custom-transition
 const greet = (name) => {
   console.log(`Hello ${name}`);
 };
 ```
 
 ```js
-// Step 4: 최종 버전
+// Step 4: the final version
 const greet = (name) => console.log(`Hello ${name}!`);
 ```
 ````
 ````
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)